x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/results/ChunkedTextEmbeddingResults.java

```diff
@@ -68,17 +100,25 @@ public String getWriteableName() {
 
     @Override
     public List<? extends InferenceResults> transformToCoordinationFormat() {
-        throw new UnsupportedOperationException("Chunked results are not returned in the coordindated action");
+        throw new UnsupportedOperationException("Chunked results are not returned in the coordinated action");
     }
 
     @Override
     public List<? extends InferenceResults> transformToLegacyFormat() {
         throw new UnsupportedOperationException("Chunked results are not returned in the legacy format");
     }
 
+    @SuppressWarnings("checkstyle:LineLength")
```

Comment:
Not sure if we need this to pass spotless but my ide was complaining because `org.elasticsearch.xpack.core.ml.inference.results.ChunkedTextEmbeddingResults.EmbeddingChunk::asMapWithListsInsteadOfArrays` was too long 🤣